LinuxFunctionsStacks.json

```diff
@@ -11,11 +11,11 @@
                 "majorVersions": [
                     {
                         "displayVersion": "3.1",
+                        "runtimeVersion": "dotnet|3.1",
                         "supportedFunctionsExtensionVersions": [
                             "~3"
                         ],
-                        "runtimeVersion": "dotnet|3.1",
```

Comment:
leave `runtimeVersion` here

Reply:
Just curious -- is there a reason these need to be ordered this way compared to the other stacks?

Reply:
never mind. I think you got it right.